lib/metrics/context.js

```diff
@@ -247,9 +227,9 @@ module.exports = function (log, config) {
   }
 }
 
-function getKey (token, event) {
-  if (token && event) {
-    return [ token.uid.toString('hex'), token.id, event ].join(':')
+function getKey (token) {
+  if (token) {
+    return [ token.uid.toString('hex'), token.id ].join(':')
```

Comment:
I'm broadly 👍 but two questions on this change.

* Will deploying this change cause a "blip" in our metrics, where flows that started against train-71 but completed against train-72 will not emit correct metrics?  If so, is there anything low-cost we could do to avoid that?  (I think a small blip is a price we can afford to pay, as long as we know up-front to expect it).

If we're changing the key anyway, is it worth making the key something like `sha256(uid, token.id)` as a defense-in-depth measure, so that any leak or compromise of the data in memcache doesn't reveal tokens or email-verification codes?

(To be clear, neither of those things are blockers, just something to think about)